SEED-platform/seed
# Fix active nav for deep URLs

#### Any background context you want to provide?
When using tabbed navigation the current tab needs to be highlighted as selected whether the current page is the default link (e.g. `/insights/custom`) or a specific id (e.g. `/insights/custom/1`)

#### What's this PR do?
Fixes nav highlight when the URL includes an ID for the account Program Setup page and the Custom Reports page

#### How should this be manually tested?
1. Browse to a specific program in the accounts Program Setup page, check that the `Program Setup` button remains highlighted
2. Browse to a specific report in Custom Reports, check that the `Custom Reports` button remains highlighted

#### What are the relevant tickets?
#3911
#3912
